lib/rubocop/cop/lint/parentheses_as_grouped_expression.rb

```diff
@@ -15,19 +15,16 @@ class ParenthesesAsGroupedExpression < Cop
 
         def on_send(node)
           _receiver, method_name, *args = *node
-          return if args.size > 1
+          return if args.size != 1
```

Comment:
This could possibly be `if !args.one?`, which will ignore potential falsey values.

```
[:foo, nil].size != 1
#=> true

![:foo. nil].one?
#=> false
```